db/structure.sql

```diff
@@ -47,6 +47,17 @@ CREATE TABLE `bmap_flowcell` (
   CONSTRAINT `fk_bmap_flowcell_to_study` FOREIGN KEY (`id_study_tmp`) REFERENCES `study` (`id_study_tmp`) ON DELETE RESTRICT ON UPDATE RESTRICT
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
+DROP TABLE IF EXISTS `cherrypicked_samples`;
```

Comment:
Why does it call `DROP TABLE`?

Reply:
Not sure. All changes to `db/structure.sql` were made as part of the `bundle exec rails db:migrate` call. Looking at the content of this file, my guess is that it attempts to drop the table to ensure there is no name conflict with the view. Note that a table with such a name did not already exist

Reply:
Cool.